@xhochy I was able to reproduce the issue


```
% MLFLOW_SKINNY=1 pip install mlflow-skinny-1.14.0.tar.gz
Processing ./mlflow-skinny-1.14.0.tar.gz
    ERROR: Command errored out with exit status 1:
     command: /Users/harutakakawamura/.pyenv/versions/miniconda3-4.7.12/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/33/kkm875tn7cxb89b8mmcykrx00000gn/T/pip-req-build-i5khje1l/setup.py'"'"'; __file__='"'"'/private/var/folders/33/kkm875tn7cxb89b8mmcykrx00000gn/T/pip-req-build-i5khje1l/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /private/var/folders/33/kkm875tn7cxb89b8mmcykrx00000gn/T/pip-pip-egg-info-5gsafbx9
         cwd: /private/var/folders/33/kkm875tn7cxb89b8mmcykrx00000gn/T/pip-req-build-i5khje1l/
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/33/kkm875tn7cxb89b8mmcykrx00000gn/T/pip-req-build-i5khje1l/setup.py", line 113, in <module>
        else open("README_SKINNY.rst").read() + open("README.rst").read(),
    FileNotFoundError: [Errno 2] No such file or directory: 'README_SKINNY.rst'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

--- a/.github/workflows/test-package-build.yml
+++ b/.github/workflows/test-package-build.yml
@@ -21,6 +21,11 @@ jobs:
         with:
           python-version: "3.7"
 
+      - name: Set environment variable for skinny client build
+        if: ${{ matrix.type == 'skinny' }}
+        run: |
+          echo "MLFLOW_SKINNY=true" >> $GITHUB_ENV
+
       - name: Build UI
         if: ${{ matrix.type == 'full' }}
         working-directory: mlflow/server/js
@@ -35,11 +40,6 @@ jobs:
       - name: Install dependencies
         run: |
           pip install wheel twine
-
-      - name: Set environment variable for skinny client build
-        if: ${{ matrix.type == 'skinny' }}
-        run: |
-          echo "MLFLOW_SKINNY=true" >> $GITHUB_ENV
 
       - name: Build distribution files
         id: build-wheel

--- a/.github/workflows/test-package-build.yml
+++ b/.github/workflows/test-package-build.yml
@@ -71,7 +71,7 @@ jobs:
 
       - name: Test installation from wheel
         run: |
-          pip install ${{ steps.build-wheel.outputs.wheel-path }}
+          pip install --force-reinstall ${{ steps.build-wheel.outputs.wheel-path }}
           python -c "import mlflow; print(mlflow.__version__)"
 
       # Anyone with read access can download the uploaded wheel on GitHub.

--- a/.github/workflows/test-package-build.yml
+++ b/.github/workflows/test-package-build.yml
@@ -36,13 +36,14 @@ jobs:
         run: |
           pip install wheel twine
 
+      - name: Set environment variable for skinny client build
+        if: ${{ matrix.type == 'skinny' }}
+        run: |
+          echo "MLFLOW_SKINNY=true" >> $GITHUB_ENV
+
       - name: Build distribution files
         id: build-wheel
         run: |
-          if [ "${{ matrix.type }}" = "skinny" ]; then
-            export MLFLOW_SKINNY="true"
-          fi
-
           # Build distribution files
           python setup.py sdist bdist_wheel
 
@@ -50,9 +51,11 @@ jobs:
           ls -lh dist
 
           # Set step outputs
+          sdist_path=$(find dist -type f -name "*.tar.gz")
           wheel_path=$(find dist -type f -name "*.whl")
           wheel_name=$(basename $wheel_path)
           wheel_size=$(stat -c %s $wheel_path)
+          echo "::set-output name=sdist-path::${sdist_path}"
           echo "::set-output name=wheel-path::${wheel_path}"
           echo "::set-output name=wheel-name::${wheel_name}"
           echo "::set-output name=wheel-size::${wheel_size}"
@@ -61,7 +64,12 @@ jobs:
         run: |
           twine check --strict ${{ steps.build-wheel.outputs.wheel-path }}
 
-      - name: Test installation
+      - name: Test installation from tarball
+        run: |
+          pip install ${{ steps.build-wheel.outputs.sdist-path }}
+          python -c "import mlflow; print(mlflow.__version__)"
+
+      - name: Test installation from wheel
         run: |
           pip install ${{ steps.build-wheel.outputs.wheel-path }}
           python -c "import mlflow; print(mlflow.__version__)"

